web/pdf_page_view.js

```diff
@@ -564,17 +575,38 @@ class PDFPageView {
       }
     }
 
+    const postponeDrawing = postponeDrawingDelay >= 0;
```

Comment:
The same comments about better delay-validation, see the `pdf_viewer.js` comments, apply here as well.